mne/channels/channels.py

```diff
@@ -502,8 +502,9 @@ def plot_sensors(self, kind='topomap', ch_type=None, title=None,
         title : str | None
             Title for the figure. If None (default), equals to ``'Sensor
             positions (%s)' % ch_type``.
-        show_names : bool
-            Whether to display all channel names. Defaults to False.
+        show_names : bool | array of str
```

Comment:
Sounds like a parameter change to `names` would be good, but I'm not sure it's worth deprecation to get there